chapters/tutorials/index.rst

```diff
@@ -0,0 +1,403 @@
+
+So we have our Flow ready with two Executors. So far it's a simple one but it is still useful to have visualize our Flow to make sure it's what we want.
+
+
+Visualize a Flow
+++++++++++++++++++++++++++++++
+
+By now, your code should look like this:
+
+.. code-block:: python
+    from jina import Flow, Document
+
+    class MyTransformer(Executor):
+        def foo(self, **kwargs):
+            print(f'foo is doing cool stuff: {kwargs}')
+
+    class MyIndexer(Executor):
+        def bar(self, **kwargs):
+            print(f'bar is doing cool stuff: {kwargs}')
+
+    f = (
+            Flow()
+            .add(uses=MyTransformer)
+            .add(uses=MyIndexer)
+        )
+
+But what if you want to visualize your Flow? you can do that with `plot`. For example:
+
+.. code-block:: python
+
+    from jina import Flow
+
+    f = (
+            Flow()
+            .add(uses=MyTransformer)
+            .add(uses=MyIndexer)
+            .plot('our_flow.svg')
+        )
+
+Let's run the code we have so far. If you try it, not much will happen since we are not indexing anything yet, but you will see the new file `our_flow.svg` created on your working folder, and if you open it you would see this:
+
+.. image:: res/plot_flow1.png
+   :width: 600
+
+You can see a Flow with two pods, but what if you have many pods? this can quickly become very messy, so it is better to name the Executors with `name='CoolName`. So in our example, we use:
+
+.. code-block:: python
+
+    from jina import Flow
+
+    f = (
+            Flow()
+            .add(name='MyTransformer', uses=MyTransformer)
+            .add(name='MyIndexer', uses=MyIndexer)
+            .plot('our_flow.svg')
+        )
+
+Now if you run this, you should have a Flow that is more explicit:
+
+.. image:: res/plot_flow2.png
+   :width: 600
+
+Use a Flow
+++++++++++++++++++++++++++++++++++++
+
+Ok, we have our Flow created and visualized. Let's put it to use now. The correct way to use a Flow is to open it as a context manager, with the `with` keyword:
+
+.. code-block:: python
+
+    with f:
+        ...
+
+Let's recap a bit what we have seen:
+
+.. code-block:: python
+
+    from jina import Flow
+    f = Flow()          # Create Flow
+
+    f.add().add()       # Add elements to Flow
+    f.plot()            # Plot a Flow
+
+    with f:             # Use Flow as a context manager
+        f.index()
+
+In our example, we have a Flow with two executors (`MyTransformer` and `MyIndexer`) and we want to use our Flow to index our data. But in this case, our data is a `csv` file, so we need to open it first
+
+.. code-block:: python
+
+    with f, open('our_dataset.csv') as fp:
+            f.index()
+
+Now we have our Flow ready, we can start to index. But we can't just pass the dataset in the original format to our Flow, we need to create a Document with the data we want to use.
```

Comment:
- "...format to our Flow, we need to..." -> "...format to our Flow. **W**e need to..."